tests/model_hub_tests/torch_tests/test_torchvision_models.py

```diff
@@ -0,0 +1,101 @@
+
+
+def get_all_models() -> list:
+    m_list = torch.hub.list("pytorch/vision")
+    m_list.remove("get_model_weights")
+    m_list.remove("get_weight")
+    return m_list
+
+
+def get_video():
+    """
+    Download video and return frames.
+    Using free video from pexels.com, credits go to Pavel Danilyuk.
+    Initially used in https://pytorch.org/vision/stable/auto_examples/plot_optical_flow.html
+    """
+    import tempfile
+    from pathlib import Path
+    from urllib.request import urlretrieve
+    from torchvision.io import read_video
+
+    video_url = "https://download.pytorch.org/tutorial/pexelscom_pavel_danilyuk_basketball_hd.mp4"
+    video_path = Path(tempfile.mkdtemp()) / "basketball.mp4"
+    _ = urlretrieve(video_url, video_path)
+
+    frames, _, _ = read_video(str(video_path), output_format="TCHW")
+    return frames
+
+
+def prepare_frames_for_raft(name, frames1, frames2):
+    w = torch.hub.load("pytorch/vision", "get_model_weights",
+                       name=name).DEFAULT
+    img1_batch = torch.stack(frames1)
+    img2_batch = torch.stack(frames2)
+    img1_batch = F.resize(img1_batch, size=[520, 960], antialias=False)
+    img2_batch = F.resize(img2_batch, size=[520, 960], antialias=False)
+    img1_batch, img2_batch = w.transforms()(img1_batch, img2_batch)
+    return (img1_batch, img2_batch)
+
+
+class TestTorchHubConvertModel(TestConvertModel):
+    def load_model(self, model_name, model_link):
+        m = torch.hub.load("pytorch/vision", model_name, weights='DEFAULT')
+        m.eval()
+        if model_name == "s3d" or any([m in model_name for m in ["swin3d", "r3d_18", "mc3_18", "r2plus1d_18"]]):
+            self.example = (torch.randn([1, 3, 224, 224, 224]),)
+            self.inputs = (torch.randn([1, 3, 224, 224, 224]),)
+        elif "mvit" in model_name:
+            # 16 frames from video
+            self.example = (torch.randn(1, 3, 16, 224, 224),)
+            self.inputs = (torch.randn(1, 3, 16, 224, 224),)
+        elif "raft" in model_name:
+            frames = get_video()
+            self.example = prepare_frames_for_raft(model_name,
+                                                   [frames[100], frames[150]],
+                                                   [frames[101], frames[151]])
+            self.inputs = prepare_frames_for_raft(model_name,
+                                                  [frames[75], frames[125]],
+                                                  [frames[76], frames[126]])
+        else:
+            self.example = (torch.randn(1, 3, 224, 224),)
+            self.inputs = (torch.randn(1, 3, 224, 224),)
+        return m
+
+    def get_inputs_info(self, model_obj):
+        return None
+
+    def prepare_inputs(self, inputs_info):
+        return [i.numpy() for i in self.inputs]
+
+    def convert_model(self, model_obj):
+        ov_model = convert_model(model_obj, example_input=self.example)
+        return ov_model
+
```

Comment:
do we need to remove cached models after execution?
For example, TF Hub does not remove it after execution so we have to remove it on our own.

Reply:
Yeah, cached models will not be deleted, need to figure out how to delete

Reply:
Done